CONTRIBUTORS.md

```diff
@@ -6182,6 +6184,8 @@
 
 - [@Mrschabs](https://github.com/Mrschabs)
 
+-[@alxPgns] (https://github.com/alxPgns)
```

Comment:
Is there a reason why you add yourself twice?
Please apply the correct syntax though:
```suggestion
- [@alxPgns](https://github.com/alxPgns)
```